generator/parseXML/ParseXML.py

```diff
@@ -79,6 +79,7 @@ def to_bool(v):
         if v is None:
             return False
         return v.lower() in ("yes", "true", "1")
+        # TODO what is returned if v.lower() is "false"?
```

Comment:
Good question. I suggest that where I don't answer a question you create a github issue for the 'question'. You could then take out the TODO.

Reply:
Now issue 15